lib/task-data/tasks/install-coreos.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     implementsTask: 'Task.Base.Os.Install',
     options: {
         // NOTE: user/pass aren't used by the coreos installer at the moment,
-        // but they are required values
+        // but they are required values for Task.Base.Os.Install
```

Comment:
From the definition of Task.Base.Os.Install, the user/password are not required (at least now).
Meanwhile, by the design of "Parameterized OS Bootstrap", no option for "username", the name for the password for root account should be "rootPassword"